.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: unit-tests
```

Comment:
Can you rename this Unit Tests please, so it is in-line with the other github workflows?